secret_store/src/key_server_cluster/jobs/consensus_session.rs

```diff
@@ -101,12 +101,23 @@ impl<ConsensusTransport, ComputationExecutor, ComputationTransport> ConsensusSes
 		})
 	}
 
-	/// Get consensus job reference.
 	#[cfg(test)]
+	/// Get consensus job reference.
```

Comment:
i think according to our style, the previous variant was correct  (attribute after doc comment)

Not sure doc comment is needed for test item anyway though